Amethyst/Managers/WindowManager.swift

```diff
@@ -622,19 +627,25 @@ extension WindowManager: ApplicationObservationDelegate {
     }
 
     func application(_ application: AnyApplication<Application>, didResizeWindow window: Window) {
-        guard userConfiguration.mouseResizesWindows() else {
+        guard
+            let screenManager: ScreenManager<WindowManager<Application>> = focusedScreenManager(),
+            let layout = screenManager.currentLayout
+        else {
+            return
+        }
+        
+        guard userConfiguration.mouseResizesWindows() || layout is FloatingGridLayout else {
             return
         }
 
         guard let screen = window.screen(), activeWindows(on: screen).contains(window) else {
             return
         }
-
-        guard
-            let screenManager: ScreenManager<WindowManager<Application>> = focusedScreenManager(),
-            let layout = screenManager.currentLayout,
-            layout is PanedLayout
-        else {
+        
+        guard layout is PanedLayout else {
+            if layout is FloatingGridLayout, case .doneDragging = mouseStateKeeper.state {
+                screenManager.setNeedsReflow(withWindowChange: .layoutChange)
```

Comment:
I don't think `.layoutChange` is right. That is for when the currently selected layout changes. We may need a different one for specifically window resizes.